packages/fxa-auth-server/test/local/routes/oauth.js

```diff
@@ -737,167 +737,6 @@ describe('/oauth/ routes', () => {
   });
 
   describe('/oauth/destroy', () => {
-    const SUCCESS = async () => {
-      return {};
-    };
-    const NOTFOUND = async () => {
-      throw error.invalidToken();
-    };
-
-    it('tries oauthdb.revokeAccessToken then oauthdb.revokeRefreshToken when told nothing about the token', async () => {
```

Comment:
I removed these because either the functionality is tested in the remote tests or not important to the correctness of the route, like which is checked first.